package.json

```diff
@@ -8,6 +8,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.30",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "@mdx-js/react": "^1.6.21",
```

Comment:
What's this package being used for?

Reply:
It's a dependency of `gatsby-interface` among others but when I did `yarn add` of a package instead of `npm install`, yarn removed it and `npm start` failed. Since declaring it here fixed the problem for me, I figured I'd leave it for future `yarn/npm` switchers not to break the project for themselves.